crates/ruff_linter/src/rules/refurb/rules/metaclass_abcmeta.rs

```diff
@@ -69,6 +74,7 @@ pub(crate) fn metaclass_abcmeta(checker: &Checker, class_def: &StmtClassDef) {
         return;
     }
 
+    let has_bases = !class_def.bases().is_empty();  
```

Comment:
nit: I think we could use something like this here:

```suggestion
    let applicability = if class_def.bases().is_empty() { Applicability::Safe } else { Applicability::Unsafe }; 
```

And then instead of the `make_fix` method, we can just use `Fix::applicable_edits` where we previously used `Fix::safe_edits`. You could then move your comments from `make_fix` here too, or possibly delete them too since I think you covered it nicely in the `Fix safety` section.